README.md

```diff
@@ -62,24 +62,23 @@ In order to test a provider resource, there are three ways to do that.
 
 1. ### Using AWS CloudFormation website
 
-   After creating or updating or deleting a stack of a provider resource in AWS Cloudformation website, you need to check on [AWS Cloudformation](https://console.aws.amazon.com/cloudformation) for events which shows a status of a resource provider.
-   
-   To see logs, after uploading a stack, it creates automatically a log for [CloudWatch](https://console.aws.amazon.com/cloudwatch) in `Log groups`.
-   
+   After creating or updating or deleting a stack of a provider resource in AWS Cloudformation website, you need to check on [AWS CloudFormation](https://console.aws.amazon.com/cloudformation) for events which show the status of a resource provider.
+      
+   To see logs, after uploading a stack, it automatically creates logs in [AWS CloudWatch](https://console.aws.amazon.com/cloudwatch) in `Log groups`.
+      
    To print a log you need to add it in code `log.Printf("")`.
    
-2.  ### Using AWS CLI Cloudformation
-    
-    After creating or updating or deleting a stack of a provider resource with AWS CLI, you can use commands to see more info like events, describe stacks and more, [see cloudformation commands](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/index.html).
-    
-    After uploading a stack, it creates automatically a log for [CloudWatch](https://aws.amazon.com/cloudwatch/).
+2.  ### Using AWS CloudFormation CLI
     
+    After creating or updating or deleting a stack of a provider resource with AWS CLI, you can use commands to see more info like events, describe stacks and more, see [CloudFormation CLI commands](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/index.html).
```

Comment:
Again slight changes: 
After creating, updating or deleting a stack of a provider resource with the AWS CLI, you can use commands to see events, describe stacks and more.  See [CloudFormation CLI commands](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/index.html) for more.